kuka_cybertech_support/urdf/kr_cybertech_ros2_control_macro.xacro

```diff
@@ -16,9 +17,6 @@
           <param name="client_port">${client_port}</param>
           <param name="controller_ip">${controller_ip}</param>
         </xacro:if>
-        <xacro:if value="${mode == 'gazebo'}">
```

Comment:
I know that currently this model does not have interias, but it should be available soon, so maybe you should add the gazebo hw if here as well